runtime/src/tests/work-item/work-item-list/page-objects/work-item-detail.page.js

```diff
@@ -28,7 +28,8 @@ class WorkItemDetailPage {
   }
 
   get workItemDetailFullPageCloseButton () {
-    return element(by.xpath(".//*[contains (@class,'pficon-close pull-right margin-right-5 padding-15 close')]"));
+//    return element(by.xpath(".//*[contains (@class,'pficon-close pull-right margin-right-5 padding-15 close')]"));
```

Comment:
@ldimaggi We can just delete this line, can't we?

Reply:
Yes!